test/py/onnx_backend_test.py

```diff
@@ -396,14 +396,7 @@ def disabled_tests_onnx_1_10_0(backend_test):
 def disabled_tests_onnx_1_11_0(backend_test):
     # errors
     # from OnnxBackendNodeModelTest
-    backend_test.exclude(r'test_gridsample_aligncorners_true_cpu')
     backend_test.exclude(r'test_gridsample_bicubic_cpu')
```

Comment:
Adding bicubic sample would complicate things further because it would require more computation steps  (e.g. tvm bicubic interpolation is more complex than the other two: https://github.com/apache/tvm/blob/main/python/tvm/topi/image/grid_sample.py#L208-L277 ), and I don't wanted to further bloat this PR, it is already around 2k line changes.
I'm happy to add it in a follow up PR.

Reply:
Created https://github.com/ROCm/AMDMIGraphX/issues/2960 to track cubic interpolation